capture.py

```diff
@@ -11,6 +11,12 @@
 from maya import cmds
 from maya import mel
 
+try:
+    from PySide2 import QtGui, QtWidgets
+except ImportError:
+    from PySide import QtGui
+    from PySide import QtGui as QtWidgets
```

Comment:
Let's make this `QtWidgets = QtGui` instead. I keep reading this as though you meant to import QtCore, or something different from the above line.